blueprints/ember-date-fns/index.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-env node */
+
+module.exports = {
+  description: 'Add `date-fns` to the consumer application',
+
+  afterInstall() {
+    return this.addPackagesToProject([
+      { name: 'date-fns', target: '^1.30.1' },
```

Comment:
Shouldn't this be a dep of `ember-date-fns` rather than added directly to the app? If we add it to the app, why would people use this addon?

Reply:
For the base helpers, I suppose. `ember-auto-import` doesn't support that.

Here is some discussion about this with @ef4: https://github.com/ef4/ember-auto-import/issues/179

Reply:
@rwwagner90 Having `date-fns` just in the addon wouldn't bring much benefit anyway.
It would actually lock you on a specific version, so this is better anyway.

In addition, this way you can just use any extra helpers you might need without having to whitelist them in some configuration file.

Reply:
@btecu I think I would suggest deprecating this addon then, as it adds no benefits, and ember-auto-import should be used.

Reply:
@rwwagner90 The two helpers that it provides are used on most projects, so it does save us from creating those helpers on every single project.

Reply:
Ah, true.